animations-tutorial.md

```diff
@@ -0,0 +1,638 @@
+---
+layout: page
+title: Animations Tutorial
+permalink: /animations-tutorial/
+---
+
+Welcome to Flutter's Animations Tutorial. You may also be interested
+in our [Animations Overview](/animations/).
```

Comment:
Speaking from experience, the Overview document is horribly confusing for beginners. Please do not encourage people to read it first.